src/lib/utils/path.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
If we keep this, then a license header is missing, but following the comment above I hope we can remove this.